src/unreal_sim/Dockerfile.dev

```diff
@@ -0,0 +1,25 @@
+FROM qutms_driverless_base:galactic
```

Comment:
id rather have the sim container be a standalone dockerfile in here, rather than having extra bases complicating our `base` pattern in the top level `docker` folder. The sim is a very standalone module in this repo, so we should isolate it as such.